articles/libraries/auth0js/index.md

```diff
@@ -85,45 +85,148 @@ Initialize a new instance of the Auth0 client as follows:
+  password: 'testpass',
+  scope: 'openid profile',
+  audience: 'urn:test'
 });
 ```
 
+### Passwordless Login
+
+Passwordless authentication allows users to log in by receiving a one-time password via email or text message. The process will require you to start the Passwordless process, generating and dispatching a code to the user, (or a code within a link), followed by accepting their credentials via the verification method. That could happen in the form of a login screen which asks for their (email or phone number) and the code you just sent them. It could also be implemented in the form of a Passwordless link instead of a code sent to the user. They would simply click the link in their email or text and it would hit your endpoint and verify this data automatically using the same verification method (just without manual entry of a code by the user).
+
+#### Start Passwordless
+
+The `passwordlessStart` method requires several options:
+
+* `connection` - a string, the connection for this authentication
+* `send` - a string, value must be either 'code' or 'link'
+
+In addition, _one_ of the two following options must be sent:
+
+* `phoneNumber` - a string containing the user's phone number for delivery of a code or link via SMS
+* `email` - a string containing the user's email for delivery of a code or link via email
+
+```js
+webAuth.passwordlessStart({
+    connection: 'Username-Password-Authentication',
+    send: 'code',
+    email: 'foo@bar.com'
+  }, function (err,res) {
+    // handle errors or continue
+  }
+);
+```
+
+#### Verify Passwordless
+
+The `passwordlessVerify` method requires several options:
+
+* `connection` - a string, the connection for this authentication
+* `verificationCode` - a string, the code sent to the user as a code or within a link.
+
+In addition, _one_ of the two following options must be sent:
+
+* `phoneNumber` - a string containing the user's phone number, to which the code or link was delivered via SMS
+* `email` - a string containing the user's email, to which the code or link was delivered via email
+
+
+```js
+webAuth.passwordlessVerify({
+    connection: 'Username-Password-Authentication',
+    email: 'foo@bar.com',
+    verificationCode: '389945'
+  }, function (err,res) {
+    // handle errors or continue
+  }
+);
+```
+
 ## Logout
 
 To logout a user, use the `logout` method. This accepts an options object, which can include a 'client_id', and a 'returnTo' URL. If you want to navigate the user to a specific URL after the logout, set that URL at the 'returnTo' parameter.
 
-::: panel-info returnTo parameter
-Note that if the 'client\_id' parameter _is_ included, the 'returnTo' URL that is provided must be listed in the **client's** "Allowed Logout URLs" in the Auth0 dashboard. 
-However, if the 'client\_id' parameter _is not_ included, the 'returnTo' URL must be listed in the "Allowed Logout URLs" at the **account level** in the Auth0 Dashboard.
-:::
+<div class="alert alert-info">
```

Comment:
Hey @jeffreylees , the idea for the alerts was to be used for short versioning information (ie This doc uses v8, for v7 refer to...), not replace panels altogether, so I would change that (and the other two in the doc) . If you think we should replace them altogether maybe we need to consult with the design team beforehand to get their opinion.